PixelHunter1995/SceneLib/IDog.cs

```diff
@@ -1,6 +1,9 @@
-﻿namespace PixelHunter1995.SceneLib
+﻿using PixelHunter1995.Utilities;
+
+namespace PixelHunter1995.SceneLib
 {
     internal interface IDog
```

Comment:
Maybe make it extend IDrawable.
Gives us a tasty z-index that can be used when interacting with overlapping dogs.
Also allows us to make all dogs (including invisible ones, which are a thing?) draw something when DEBUG == true